crml/sylo/src/migration.rs

```diff
@@ -55,7 +55,7 @@ impl<T: Trait> Module<T> {
 	// Ensure `origin` is an authorized Sylo data migrator
 	pub fn ensure_sylo_migrator(origin: T::Origin) -> DispatchResult {
 		let account_id = ensure_signed(origin)?;
-		ensure!(AuthorisedMigrators::<T>::contains_key(account_id), BadOrigin);
+		ensure!(AuthorisedMigrators::<T>::get(account_id), "NotASyloMigrator");
```

Comment:
I suppose `get(...)` returns Some/None. So I wonder how we are ensuring the boolean evaluation of an option?

Reply:
Good q, internally `::get` will call the following:
```
from_optional_value_to_query(v: Option<bool>) -> Self::Query {
    v.unwrap_or_else(|| Default::default())
}
```
So it returns the type's default rather than `None`.

 You can check this by running `cargo expand`